src/de/jost_net/JVerein/gui/dialogs/BuchungUebernahmeProtokollDialog.java

```diff
@@ -92,6 +92,7 @@ public String format(Object o)
     bu.addColumn("Datum", "datum",
         new DateFormatter(new JVDateFormatTTMMJJJJ()));
     bu.addColumn("Name", "name");
+    bu.addColumn("IBAN oder Kontonummer", "iban");
```

Comment:
Der Kommentar hängt nur hier, weil GitHub keinen Kommentar an unveränderten Dateien zulässt.

Buchungen können auf zwei Wegen importiert werden: aus Hibiscus und aus CSV-Dateien. Im Moment kann die IBAN beim CSV-Import nicht gesetzt werden, das sollte ergänzt werden.

Reply:
Muss das in diesem PR gemacht werden, oder kann die Erweiterung auf den nächsten Issue/PR verschoben werden?

Reply:
Kann natürlich auch verschoben werden.